orchestrator/process/utils.go

```diff
@@ -25,29 +25,37 @@ func buildStagedProcessesList(managerConfig config.ProcessesManagerConfig) []pro
 	var stagedProcessesList []process
 
 	for _, group := range managerConfig.ProcessGroups {
-		groupArgs := getProcessGroupArgs(group.Script, group.ArgPrefix, group.ArgAssign, group.Args)
+		groupArgs := getProcessArgs(group.Script, group.ArgPrefix, group.ArgAssign, group.Args.GroupArgs)
 		groupInstance := newProcessGroup(group.Name, group.Replicas, group.Command, groupArgs)
 
 		for i := 0; i < group.Replicas; i++ {
-			stagedProcessesList = append(stagedProcessesList, newProcess(groupInstance))
+			processArgs := []string{}
```

Comment:
Can we add the script here instead of passing it to the `getProcessArgs` function?